### ISSUE TITLE
[BUG]Downstream clusters "cluster tools not available" after upgrading Rancher to v2.8.3 in Air Gap Environment

### ISSUE THREAD

**Issue by shindebshekhar** [rancher/rancher#45240]
**Rancher Server Setup**
- Rancher version: v2.8.3
- Installation option (Docker install/Helm Chart): Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):
- Proxy/Cert Details:
Air-Gap Environment
**Information about the Cluster**
- Kubernetes version: v1.27.10+rke2r1

**Describe the bug**
After upgrading Rancher from v2.8.2 to v.2.8.3 the cluster tools are not available on Downstream clusters. On further checking found that the rancher repositories(https://git.rancher.io) are not able to sync. On Management cluster the charts are available

Below is the error:
Rancher UI-->Apps-->Charts --> Repostories
`Ensure failure: git -C /var/lib/rancher-data/local-catalogs/v2/rancher-partner-charts/8f17acdce9bffd6e05a58a3798840e408c4ea71783381ecd2e9af30baad65974 fetch origin -- 8a753a4d6549a40e6eb725151cbbcc7e9065fce7 error: exit status 128, detail: fatal: unable to access 'https://git.rancher.io/partner-charts/':`

On Rollback to v2.8.2 the charts are in sync and cluster tools is also available on all downstream clusters

**Comment by Woovie** [comment]
Resolved this issue today in 2.8.3, the problem was that `setting.management.cattle.io/system-catalog` should have values set for `source` and `value`.

Example bad configuration:

```
> kubectl describe setting.management.cattle.io/system-catalog
...
Source:
Value:
...
```

And when fixed:
```
> kubectl describe setting.management.cattle.io/system-catalog
...
Source:                env
Value:                 bundled
...
```

Use a simple `kubectl edit setting.management.cattle.io/system-catalog` to modify the settings object and correct this. Make sure you specify the entire API resource name, as `settings` are a commonly used API resource across several Rancher namespaces, such as Longhorn.

**Comment by mh013370** [comment]
> Resolved this issue today in 2.8.3, the problem was that `setting.management.cattle.io/system-catalog` should have values set for `source` and `value`.
> 
> Example bad configuration:
> 
> ```
> > kubectl describe setting.management.cattle.io/system-catalog
> ...
> Source:
> Value:
> ...
> ```
> 
> And when fixed:
> 
> ```
> > kubectl describe setting.management.cattle.io/system-catalog
> ...
> Source:                env
> Value:                 bundled
> ...
> ```
> 
> Use a simple `kubectl edit setting.management.cattle.io/system-catalog` to modify the settings object and correct this. Make sure you specify the entire API resource name, as `settings` are a commonly used API resource across several Rancher namespaces, such as Longhorn.

This does work, but it's worth clarifying this edit needs to be performed on the _downstream_ clusters and not the Rancher management cluster. The latter has no effect.

I assume this is something that ought to be fixed in Rancher itself?

**Comment by Woovie** [comment]
> > Resolved this issue today in 2.8.3, the problem was that `setting.management.cattle.io/system-catalog` should have values set for `source` and `value`.
> > Example bad configuration:
> > ```
> > > kubectl describe setting.management.cattle.io/system-catalog
> > ...
> > Source:
> > Value:
> > ...
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > And when fixed:
> > ```
> > > kubectl describe setting.management.cattle.io/system-catalog
> > ...
> > Source:                env
> > Value:                 bundled
> > ...
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Use a simple `kubectl edit setting.management.cattle.io/system-catalog` to modify the settings object and correct this. Make sure you specify the entire API resource name, as `settings` are a commonly used API resource across several Rancher namespaces, such as Longhorn.
> 
> This does work, but it's worth clarifying this edit needs to be performed on the _downstream_ clusters and not the Rancher management cluster. The latter has no effect.
> 
> I assume this is something that ought to be fixed in Rancher itself?

Yep, this is a bug still. Glad it did help, and thanks I did forget to clarify that it's for downstream. I'll edit my comment.

**Comment by vrevelas** [comment]
same issue here, but the error I was seeing on the Apps -> Repositories page was `Update failure: Repo [https://git.rancher.io/partner-charts] is not accessible: Get "https://git.rancher.io/repos/partner-charts/commits/main": context deadline exceeded (Client.Timeout exceeded while awaiting headers)`. @Woovie 's workaround worked, thanks!

**Comment by Skybladev2** [comment]
I restarted Rancher fleet-agent and fleet-controller deployments for these settings to take effect and it also worked in my case.
